docker/list_operator.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# -x echos all lines for debug
+# set -x
+
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+# shellcheck source=./helpers.sh
+# shellcheck disable=SC1091
+source "$SCRIPT_DIR"/helpers.sh
+
+if [ "$DEPLOY_ENV" = "TESTNET" ]; then
+    LOCAL_ETHEREUM_RPC_URL="$TESTNET_RPC_URL"
+else
+    LOCAL_ETHEREUM_RPC_URL=${LOCAL_ETHEREUM_RPC_URL:-http://localhost:8545}
+fi
+
+# Read deployment info
+# ECDSAStakeRegistryAddress=$(cat /root/.nodes/avs_deploy.json | jq -r '.addresses.stakeRegistryImpl')
+ECDSAStakeRegistryAddress=$(cat /root/.nodes/avs_deploy.json | jq -r '.addresses.stakeRegistry')
+if [ -z "$ECDSAStakeRegistryAddress" ]; then
+    echo "Error: failed to read ECDSAStakeRegistry from /root/.nodes/avs_deploy.json"
+    exit 1
+fi
+
+echo "=== ECDSA Stake Registry Status ==="
+echo "Contract Address: $ECDSAStakeRegistryAddress"
+
+# Get total weight and threshold
+echo -e "\n=== Quorum Information ==="
+TOTAL_WEIGHT=$(cast call "$ECDSAStakeRegistryAddress" "getLastCheckpointTotalWeight()(uint256)" --rpc-url "$LOCAL_ETHEREUM_RPC_URL")
+THRESHOLD_WEIGHT=$(cast call "$ECDSAStakeRegistryAddress" "getLastCheckpointThresholdWeight()(uint256)" --rpc-url "$LOCAL_ETHEREUM_RPC_URL")
+echo "Total Weight: $TOTAL_WEIGHT"
+echo "Threshold Weight: $THRESHOLD_WEIGHT"
+
+# Get current block height and calculate range
+LATEST_BLOCK=$(cast block-number --rpc-url "$LOCAL_ETHEREUM_RPC_URL")
+FROM_BLOCK=$((LATEST_BLOCK - 2000))
```

Comment:
I'm assuming this is always valid for our targets... but if hitting some young chain for some reason, could be problematic? not a blocker imho